### PR TITLE
Consistent ui behavior in send page

### PR DESCRIPTION
**Pull request by Nasicus**
I found it very annoying that in the "address" field sometimes you only had the address and sometimes only the contact / wallet label.
Plus it wasn't nice that the field jumped around so much (hidden / displayed).
Also you could create a new contact for a wallet instead of just setting a label.

I changed it so it's more consistent for the user and also should be more readable & understandable in code.

Basically there are now 5 possible states in the ui concerning this field:

1. Initial (no address)
init

2. Existing Contact
existing_contact

3. Wallet with label
existing_wallet

4. Wallet without label
wallet_no_label

5. Unknown (new) address
unknown_contact

I made the field disabled for existing contacts & labels.

Todo:

-  [x]  wait until #115 is merged back and use method from there to set label

### CODE REVIEW

**Comment by Nasicus** [comment]
This one is ready for review now ;)

**Comment by luciorubeens** [comment]
Wow, good job! :+1:
